build-info-extractor-gradle/src/test/java/org/jfrog/gradle/plugin/artifactory/Utils.java

```diff
@@ -243,6 +245,24 @@ private static void checkBuildInfoModules(Build buildInfo, int expectedModules,
         }
     }
 
+    /**
+     * Check expected properties is found on each published artifacts.
```

Comment:
Check expected properties is found on each published artifacts.
-->
Check that the expected properties are found on each published artifact.